docs/reference/token-scopes.md

```diff
@@ -2,6 +2,9 @@
 
 ## Fine grained access token scopes
 
+**Note**: The user who owns the token must have admin access to the organisation or repository
+(so having a token with the necessary permissions is not enough).
```

Comment:
I think this is a really useful note to include here. As a nitpick, I would rearrange the order of the statements, maybe something like the following:

**Note**: In addition to having a token with the necessary permissions, the user who owns the token also must have admin access to the organisation or repository.